internal/archive/module_location.go

```diff
@@ -22,6 +21,15 @@ func (ep *ModuleLoc) GetTargetTmpDir() string {
 	return ep.loc.GetTarget()
 }
 
+// GetTargetTmpDir - gets the target root path;
```

Comment:
Change the comment (it should be `GetTargetTmpRoot`, and remove the `;` at the end)

Reply:
done